deployments/helm-chart/values.yaml

```
@@ -280,6 +280,9 @@ controller:
     ## Set the port where the readiness endpoint is exposed.
     port: 8081
 
+  ## Enable collection of latency metrics for upstreams. Requires enable-prometheus-metrics
```

In helm values file if you grep around for "Requires" you'll see that we reference fields by where they are in the yaml structure. 

This makes it easier to find other fields in the same file where they have differing names between the yaml and the cli arg, and lets people copy and paste directly to a `--set` command, for example: `helm install --set prometheus.create=true`.

```suggestion
  ## Enable collection of latency metrics for upstreams. Requires prometheus.create.
```